Makefile

```diff
@@ -220,21 +220,17 @@ release-ios: export TARGET := ios
 release-ios: export BUILD_ENV ?= prod
 release-ios: watchman-clean ##@build Build release for iOS release
 	@git clean -dxf -f target/ios && \
-	$(MAKE) jsbundle-ios && \
+	$(MAKE) jsbundle && \
 	xcodebuild -workspace ios/StatusIm.xcworkspace -scheme StatusIm -configuration Release -destination 'generic/platform=iOS' -UseModernBuildSystem=N clean archive
 
-jsbundle-android: SHELL := /bin/sh
-jsbundle-android: export TARGET := android
-jsbundle-android: export BUILD_ENV ?= prod
-jsbundle-android: ##@jsbundle Compile JavaScript and Clojurescript into app directory
-	# Call nix-build to build the 'targets.mobile.android.jsbundle' attribute and copy the.js files to the project root
-	nix/scripts/build.sh targets.mobile.android.jsbundle && \
-	mv result/*.js ./
-
-jsbundle-ios: export TARGET := ios
-jsbundle-ios: export BUILD_ENV ?= prod
-jsbundle-ios: ##@jsbundle Compile JavaScript and Clojure into index.ios.js
-	yarn shadow-cljs release mobile
+jsbundle: SHELL := /bin/sh
+jsbundle: export BUILD_ENV ?= prod
+jsbundle: ##@jsbundle Compile JavaScript and Clojurescript to create index.js and app.
+	nix/scripts/build.sh targets.mobile.jsbundle
+
+# Support for legacy naming
+jsbundle-ios: jsbundle
```

Comment:
@jakubgs Just curious, why we are adding backward compatibility? Are we using this in workflows or something?

Reply:
so @ajayesivan  how this works is you can now run the following make command like this `make jsbundle-ios` and `make jsbundle-android` to generate the respective jsbundles.

These make commands are used in CI also i believe.

Reply:
They are not used anywhere. I added them just in case someone complains. I'm happy to remove them.